packages/@aws-cdk/integ-tests-alpha/lib/assertions/providers/sdk-api-metadata.json

```diff
@@ -263,7 +263,7 @@
     "cors": true
   },
   "iotdata": {
-    "prefix": "iot-data",
+    "prefix": "iot",
```

Comment:
Where does this file come from? Don't we think that a future update will overwrite this value back again? 

Why is the current value causing a problem?